presto-main/src/main/java/com/facebook/presto/sql/NodeUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql;
+
+import com.facebook.presto.sql.tree.OrderBy;
+import com.facebook.presto.sql.tree.SortItem;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Optional;
+
+public class NodeUtils
+{
+    private NodeUtils() {}
+
+    public static List<SortItem> getSortItemsFromOrderBy(Optional<OrderBy> orderBy)
```

Comment:
There are more nodes with this method to be added if we wanted to follow this approach, i.e.: `QuerySpecification`, `Query`, `Window`.
I think it's not worth duplication.